deploy-manage/security/self-auto-setup.md

```diff
@@ -0,0 +1,149 @@
+---
+navigation_title: Automatic security setup
+applies_to:
+  self: ga
+mapped_pages:
+  - https://www.elastic.co/guide/en/elasticsearch/reference/current/configuring-stack-security.html
+---
+
+% Scope: Automatic setup
+% Original title: Start the Elastic Stack with security enabled automatically
+# Automatic security setup [configuring-stack-security]
```

Comment:
might want to say that these steps are customized for different package types in the install docs and link to them.

thinking of things like storing the ES password as an env variable that differ between operating systems.

Reply:
I'll add a small note and link for the install docs. The problem is that if anyone goes to the install doc then they have the same guide almost :-D
This guide has some things without much sense, but I'd prefer to improve that at a later stage.